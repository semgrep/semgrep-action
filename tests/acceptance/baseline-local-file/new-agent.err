=== Running: SEMGREP_BASELINE_COMMIT="HEAD~1" SEMGREP_RULES=".semgrep.yml" semgrep ci --no-suppress-errors
Running `semgrep ci` without API token but with configs ('.semgrep.yml',)


┌────────────────┐
│ Debugging Info │
└────────────────┘

  SCAN ENVIRONMENT
  versions    - semgrep x.y.z on python x.y.z
  environment - running in environment git, triggering event is unknown


┌─────────────┐
│ Scan Status │
└─────────────┘

Fetching configuration from semgrep.dev
Scanning 1 file.
  Current version has 1 finding.

Switching repository to baseline commit 'HEAD~1'.
  Will report findings introduced by these commits (may be incomplete for shallow checkouts):
    * aaaaaaa add rules
  The current branch is missing these commits from the baseline branch:

  Any finding these commits fixed will look like a new finding in the current branch.
  To avoid reporting such findings, compare to the branch-off point with:
    --baseline-commit=$(git merge-base HEAD~1 HEAD)

Scanning 1 file.


┌─────────┐
│ Results │
└─────────┘



┌──────────────┐
│ Scan Summary │
└──────────────┘

Some files were skipped or only partially analyzed.
  Scan was limited to files changed since baseline commit.

CI scan completed successfully.
  Found 1 finding (1 blocking) from 1 rule.
  Has findings for blocking rules so exiting with code 1
