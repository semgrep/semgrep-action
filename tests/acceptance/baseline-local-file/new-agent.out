

┌─────────────────────────┐
│ 1 Blocking Code Finding │
└─────────────────────────┘

    bar.py
       python.lang.correctness.useless-eqeq.useless-eqeq
          This expression is always True: `3 == 3` or `3 != 3`. If testing for
  floating point NaN, use
          `math.isnan(3)`, or `cmath.isnan(3)` if the number is complex.
          Details: https://sg.run/GeAp

            1┆ 3 == 3

  BLOCKING CODE RULES FIRED:
    python.lang.correctness.useless-eqeq.useless-eqeq

